### PR TITLE
[OpenCV/zh-cn]Add translation for opencv

### PR DESCRIPTION
**Pull request by GengchenXU**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by GengchenXU** [comment]
translate intoCHinese

**Comment by imba-tjd** [comment]
> Pull request title is prepended with [language/lang-code]

**Comment by mxschumacher** [comment]
Hey @imba-tjd - could you please review this translation? I don't speak Mandarin and have a hard time verifying this PR

File: zh-cn/opencv-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+# 显示图片
+# imshow() 函数被用来显示图片
+cv2.imshow('Image',img)
+# 第一个参数是窗口的标题，第二个参数是image
+# 如果你得到错误，对象类型为None，你的图像路径可能是错误的。请重新检查图像包
+cv2.waitKey(0)
+# waitKey() 是一个键盘绑定函数，参数以毫秒为单位。对于GUI事件，必须使用waitKey()函数。
+
+# Writing an image
```
Comment (imba-tjd) [inline zh-cn/opencv-cn.html.markdown:37]: 这个可以翻一下嘛，保存图片呗。

Comment (GengchenXU) [reply zh-cn/opencv-cn.html.markdown:37]: 我修改了

Comment (imba-tjd) [reply zh-cn/opencv-cn.html.markdown:37]: This has been merged. forget it. 算了吧

Comment (mxschumacher) [reply zh-cn/opencv-cn.html.markdown:37]: Is something wrong?

Comment (imba-tjd) [reply zh-cn/opencv-cn.html.markdown:37]: @mxschumacher I gave a suggestion to an untranslated entry. You merged this before the author modified the file. But it's not so important.

**Pull request merged**